test/unit/org/apache/cassandra/cql3/EmptyValuesTest.java

```diff
@@ -76,6 +78,11 @@ private void verify(String emptyValue) throws Throwable
             try (PrintStream out = new PrintStream(buf, true))
             {
                 ProcessBuilder pb = new ProcessBuilder("tools/bin/sstabledump", ssTable.getFilename());
+                Map<String, String> envs = pb.environment();
+                envs.put("JVM_OPTS", "-D" 
+                                     + CassandraRelevantProperties.SSTABLE_FORMAT_TTL_MODE.getKey()
+                                     + "="
+                                     + CassandraRelevantProperties.SSTABLE_FORMAT_TTL_MODE.getString());
```

Comment:
What's the purpose of this?

Reply:
Without it sstabledump wouldn't be called with the right feature flag value.

Reply:
I see in your latest proposed commit you removed that. The test will fail if you do so with `test-oa`, it's needed.